cmd/mattermost/commands/channel.go

```diff
@@ -343,7 +343,7 @@ func moveChannelsCmdF(command *cobra.Command, args []string) error {
 
 	username, erru := command.Flags().GetString("username")
 	if erru != nil || username == "" {
-		return errors.New("Username is required")
+		return errors.New("Username is required. Usage --username user-1.")
```

Comment:
Is `Usage ...` necessary here, or is that implied with the other changes.

Reply:
we can remove 0/5

Reply:
done